defaults/Job.dhall

```diff
@@ -13,6 +13,7 @@ let Strategy = ../types/Strategy.dhall
 in  { name = None Text
     , needs = None (List Text)
     , strategy = None Strategy
+    , environment = None Text
```

Comment:
we could only support the following extended case. wdyt? we could have a JobEnv Type.

```
    environment: 
      name: production
      url: https://github.com
```